Makefile

```diff
@@ -32,6 +35,20 @@ ut: dist/calico_kubernetes
 	'/tmp/etcd -data-dir=/tmp/default.etcd/ >/dev/null 2>&1 & \
 	PYTHONPATH=/code/calico_kubernetes nosetests calico_kubernetes/tests -c nose.cfg'
 
+# UT runs on Cicle need to create the calicoctl binary
```

Comment:
I don't think UT runs on Circle need to create the calicoctl binary for the calico_kubernetes repo.